pkg/adaptor/proxy/service.go

```diff
@@ -94,16 +102,20 @@ func (s *proxyService) getImageName(annotations map[string]string) (string, erro
 func (s *proxyService) CreateContainer(ctx context.Context, req *pb.CreateContainerRequest) (*types.Empty, error) {
 
 	logger.Printf("CreateContainer: containerID:%s", req.ContainerId)
-	if len(req.OCI.Annotations) > 0 {
-		logger.Print("    annotations:")
-		for k, v := range req.OCI.Annotations {
-			logger.Printf("        %s: %s", k, v)
-		}
-	}
 	if len(req.OCI.Mounts) > 0 {
 		logger.Print("    mounts:")
 		for _, m := range req.OCI.Mounts {
 			logger.Printf("        destination:%s source:%s type:%s", m.Destination, m.Source, m.Type)
+
+			if isNodePublishVolumeTargetPath(m.Source, kataDirectVolumesDir) {
+				req.OCI.Annotations[volumeTargetPathKey] = m.Source
```

Comment:
What if multiple mount path in a single container?

Reply:
Multiple mount paths are not considered in this PR. We can figure out how to support multiple mount paths in new issues, I think.

Reply:
Looks not too complex to support multiple mounts? How about set `req.OCI.Annotations[volumeTargetPathKey]` to a strings array or a string separated by `,` to represents a string array?

Reply:
And I agree we can take one mount path in one containers as a start if it's too complex to support multiple mounts.